programs/survival/what_biome_on_block.sc

```diff
@@ -0,0 +1,79 @@
+// shows biomes blockwise. useful for marking out rivers, beaches, soul sand valley and stuff.
+// now works across diemsions (yes even in the_end)
+__config()->{
+    'stay_loaded'->true,
+    'scope' -> 'global',
+    'commands' -> {
+        '' -> _() -> print(format('r Missing arguments. Correct syntax /showbiomes <toggle>')),

```

Comment:
Right now this is not true, the name of the app is "what_biome_on_block", hence the base command is also that. You can use `system_info('app_name')` to get the app's name to fill here.

Reply:
ah right. I missed it. Showbiome was the original name for the app. But, it was not descriptive enough imo. Therefore I changed the name to be more descriptive and missed changing it here

Reply:
I like show biomes, but my point was, since the base command is decided by the app's (file) name, any user can change it to whatever they want. To accomodate for that, `system_info('app_name')` returns the current app name as a string so you can use it in messages like this one.